chapter04/qli/ex5.pl

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/perl
+
+use warnings;
+use strict;
+
+my @names = ();
+
+sub greet {
+  my ( $name ) = @_;
+
+  if ( @names == 0 ){
```

Comment:
You don't have to have to use `== 0`, you can just use `if ( @names )` to determine if the array has elements in it or not.